src/main/java/mate/academy/internet/shop/controller/AddItemToBucketController.java

```diff
@@ -25,7 +25,8 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         Long userId = (Long) req.getSession().getAttribute("userId");
         Bucket bucket = bucketService.getByUserId(userId);
         String itemId = req.getParameter("item_Id");
-        Item item = itemService.get(Long.valueOf(itemId));
+        Item item = null;
+        item = itemService.get(Long.valueOf(itemId));
```

Comment:
what's the reason of this split?

Reply:
sorry, result of unintended and deleted try/catch block